documents/WCA Bylaws.md

```diff
@@ -20,31 +20,69 @@ The name of this corporation is World Cube Association (“WCA”).
 
-### 2.2	Annual Dues
-The amount required for annual dues shall be $0 each year, unless changed by a majority vote of the members at a properly announced meeting. Continued membership is contingent upon being up-to-date on membership dues.
-
-### 2.3	Rights of Members
-Each member shall be eligible to appoint in writing one voting representative to cast the member’s vote in WCA elections.
-
-### 2.4	Resignation and Termination
+### 2.2 Types of Membership
+The WCA shall offer the following type of memberships to its organization:
+  1. Regional Organizations;
+  2. Associate Regional Organizations;
+  3. Individual voting members;
+  4. Individual non-voting members.
+The Board reserves the right to establish or remove membership categories.  
+  
+### 2.3 Regional Organization Membership
+Regional Organization membership is available for registered entities who act to organize WCA activities within their specified regions. Applications for this type of membership are to be made in writing to the Board of Directors provided the entity meets the following requirements:
+  1. The Regional Organization has a name, address and contact details;
+  2. The Regional Organization has a website and a logo;
+  3. The Regional Organization provides a current copy of it’s Bylaws to the WCA;
+  4. The Regional Organization is registered as a legal entity in the area in which they operate;
+  5. The Regional Organization provides an description of their organizational structure;
+  6. The Regional Organization provides a clear description of the area they cover along with a report on past activities proving they act as the entity which organises WCA activities within their region.
+This membership is active from the moment they are acknowledged by the Board of Directors as meeting the requirements for this category and continues until this membership is revoked either through resignation or termination. This category of members do not have the right to vote in matters of the WCA and any elections.
+
+### 2.4 Associate Regional Organization Membership
+Associate Regional Organization membership is available for groups who act to organize WCA activities within their specified regions but however, are not currently a registered entity within the region which they operate. Applications for this type of membership are to be made in writing to the Board of Directors provided the entity meets the following requirements:
+  1. The Regional Organization has a name, address and contact details;
+  2. The Regional Organization has a website and a logo;
+  3. The Regional Organization provides a current copy of it’s Bylaws to the WCA;
+  5. The Regional Organization provides an description of their organizational structure;
```

Comment:
a description, not an description

Reply:
Thank you for your feedback these have all been fixed